doc/api/buffer.markdown

```diff
@@ -638,9 +609,9 @@ console.log(buf.length);
 ### buf.readDoubleBE(offset[, noAssert])
 ### buf.readDoubleLE(offset[, noAssert])
 
-* `offset` Number
-* `noAssert` Boolean, Optional, Default: false
-* Return: Number
+* `offset` {Number} `0 <= offset <= buf.length`
```

Comment:
Shouldn't it actually be `0 <= offset < buf.length`? Anyways, I'd reduce the description to just "zero-based byte offset" or similar.